cg/cli/upload/scout.py

```diff
@@ -56,4 +58,46 @@ def scout(context, re_upload, print_console, case_id):
         )
         context.abort()
 
-    scout_api.upload(scout_config, force=re_upload)
+    context.invoke(upload_case_to_scout, case_id=case_id, re_upload=re_upload)
+
+
+@click.command()
+@click.option("-r", "--re-upload", is_flag=True, help="re-upload existing analysis")
+@click.option("--dry-run", is_flag=True)
+@click.argument("case_id")
+@click.pass_context
+def upload_case_to_scout(context, re_upload, dry_run, case_id):
+    """Upload variants and case from analysis to Scout."""
+
+    click.echo(click.style("----------------- CONFIG -----------------------"))
+
+    def _get_load_config_from_hk(hk_api: hk.HousekeeperAPI, case_id):
+        tag_name = "scout-load-config"
```

Comment:
Should this be define as a global constant instead and imported here? Will make it easier to remember to update this tag if it is defined in only one place in cg. If the tag names in HK will change in the future.

Reply:
oh, now I see that you have made this sub instead: `get_load_config_tag`. Could you use that one here as well?

Reply:
Fixed according to the suggestion 👍